src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/EndsWith.txt

```diff
@@ -142,13 +142,13 @@ true
 >> EndsWith(Blank(),"ok")
 false
 
-//Null end case
+//Null end case - coerced to ""
 >> EndsWith("Hello",Blank())
-false
+true
 
-//Null both case
+//Null both case - both coerced to ""
```

Comment:
Could you add another case here that verifies the coercion- EndsWith("", Blank())?

Reply:
Added here and in StartsWith as well